annotationPipeline/src/main/java/org/cbioportal/annotation/pipeline/MutationRecordReader.java

```diff
@@ -85,9 +85,9 @@ public void open(ExecutionContext ec) throws ItemStreamException {
         processComments(ec, genomeNexusVersion);
         List<MutationRecord> mutationRecords = loadMutationRecordsFromMaf();
         if (postIntervalSize > 0) {
-            this.allAnnotatedRecords = annotator.getAnnotatedRecordsUsingPOST(summaryStatistics, mutationRecords, isoformOverride, replace, postIntervalSize);
+            this.allAnnotatedRecords = annotator.getAnnotatedRecordsUsingPOST(summaryStatistics, mutationRecords, isoformOverride, replace, postIntervalSize, true);
```

Comment:
This is preserving the ORIGINAL logic which was accidentally lost. We always wanted to re-annotate when we run genome-nexus-annotation-pipeline jar from the command line.
Genome nexus annotation pipeline does not have a command line flag for forcing re-annotation. Instead it has the replace flag for replacing HUGO and Entrez gene Ids.

Logic here:
https://github.com/genome-nexus/genome-nexus-annotation-pipeline/pull/108/files#diff-3d8ddf8f301d85f3990b5327a09a79f7R188